templates/index.html

```diff
@@ -41,14 +50,36 @@ <h2>Select the right display server for you</h2>
   </div>
   <div class="row">
     <div class="col-4 p-card">
-        <a class="p-card__link-title" href="/doc/"><img class="p-card__thumbnail" src="https://assets.ubuntu.com/v1/8c0acddd-Mir.svg" alt="Mir Server wordmark"/></a>
+        <a class="p-card__link-title" href="/doc/">
+          {{
+            image (
+            url="https://assets.ubuntu.com/v1/b93c19ab-Canonical-Mir-2022.svg",
+            alt="",
+            width="61",
+            height="36",
+            hi_def=True,
+            loading="auto"
```

Comment:
```suggestion
            loading="lazy"
```
can be lazy as it isn't near the top of the page